Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -418,7 +418,7 @@ private final class TokenStreamCreator: SyntaxVisitor {
     // line because continuation breaks don't stack. There are no breaks around the first condition
     // because if-statements look better without a break between the "if" and the first condition.
     for condition in node.conditions.dropFirst() {
-      before(condition.firstToken, tokens: .break(.open, size: 0))
+      before(condition.firstToken, tokens: .break(.open(kind: .continuation), size: 0))
```

Comment:
The comment above this loop probably needs to be updated now—in particular, the part about "so that those conditions have at least 1 _non-continuation based indent_ from an indentation stack".

Reply:
Doh. Good call. I updated the similar comment for guard statements too.